token/cli/src/main.rs

```diff
@@ -2650,81 +3048,88 @@ fn main() {
-        if let Some((minimum_balance_for_rent_exemption, instruction_batches)) = transaction_info {
-            let fee_payer = Some(&config.fee_payer);
-            let signer_info = CliSignerInfo {
-                signers: bulk_signers,
-            };
-
-            for instructions in instruction_batches {
-                let message = if let Some(nonce_account) = config.nonce_account.as_ref() {
-                    Message::new_with_nonce(
-                        instructions,
-                        fee_payer,
-                        nonce_account,
-                        config.nonce_authority.as_ref().unwrap(),
-                    )
-                } else {
-                    Message::new(&instructions, fee_payer)
-                };
-                let (recent_blockhash, fee_calculator) = config
-                    .blockhash_query
-                    .get_blockhash_and_fee_calculator(
-                        &config.rpc_client,
-                        config.rpc_client.commitment(),
-                    )
-                    .unwrap_or_else(|e| {
-                        eprintln!("error: {}", e);
-                        exit(1);
-                    });
-
-                if !config.sign_only {
-                    check_fee_payer_balance(
-                        &config,
-                        minimum_balance_for_rent_exemption + fee_calculator.calculate_fee(&message),
-                    )?;
-                }
-
-                let signers = signer_info.signers_for_message(&message);
-                let mut transaction = Transaction::new_unsigned(message);
-
-                if config.sign_only {
-                    transaction.try_partial_sign(&signers, recent_blockhash)?;
-                    println!(
-                        "{}",
-                        return_signers_with_config(
-                            &transaction,
-                            &config.output_format,
-                            &ReturnSignersConfig {
-                                dump_transaction_message: config.dump_transaction_message,
-                            }
-                        )?
-                    );
-                } else {
-                    transaction.try_sign(&signers, recent_blockhash)?;
-                    let signature = if no_wait {
-                        config.rpc_client.send_transaction(&transaction)?
-                    } else {
-                        config
-                            .rpc_client
-                            .send_and_confirm_transaction_with_spinner(&transaction)?
-                    };
-                    let signature = CliSignature {
-                        signature: signature.to_string(),
-                    };
-                    println!("{}", config.output_format.formatted_string(&signature));
-                }
-            }
-        }
-        Ok(())
-    })
     .map_err(|err| {
         eprintln!("{}", err);
         exit(1);
     });
```

Comment:
```suggestion
    })?;
    println!("{}", result);
```
where result is defined on L2635: `let result = match (sub_command, sub_matches) {`